configs/cms-xcache/config.d/50-cms-xcache-authz.cfg

```diff
@@ -0,0 +1,13 @@
+#
+# Configure security for cache servers of the CMS data federation
+#
+# ************************************************************
+# * NOTE: Changes to this file will be PRESERVED on upgrades *
```

Comment:
Is this true? Files in the 50s are package-owned so it shouldn't be.

Reply:
fixed